src/Fmt_ast.ml

```diff
@@ -743,7 +743,7 @@ and fmt_core_type c ?(box = true) ?(in_type_declaration = false) ?pro
                      $ fmt_core_type c (sub_typ ~ctx typ) )
                  $ fmt_docstring_padded c doc
                  $ fmt_attributes c ~pre:(str " ") ~key:"@" atrs )
-        | Oinherit typ -> fmt_core_type c (sub_typ ~ctx typ)
+        | {pof_desc= Oinherit typ; _} -> fmt_core_type c (sub_typ ~ctx typ)
```

Comment:
We probably need to handle `pof_loc` and `pof_attributes` in the `Oinherit` case

Reply:
Fixed. However I could not find the syntax for this.

Reply:
```
type t = < a : int >
type u = < (t [@test]); b : int> 
```

Reply:
This is working fine on master, because the attribute is attached on the type with ocaml 4.08